contracts/RRUtils.sol

```diff
@@ -152,4 +152,71 @@ library RRUtils {
 
         return false;
     }
+
+    function compareLabel(bytes memory self, bytes memory other) internal pure returns (int){
+        uint sLength = labelCount(self, 0);
+        uint oLength = labelCount(other, 0);
+        uint shortest;
+
+        if (sLength < oLength){
```

Comment:
You can skip an `if` by setting `shortest = sLength`, then just checking the other case, like in `compare`.